test/e2e/storage/csi_objects.go

```diff
@@ -37,12 +38,30 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-const (
-	csiHostPathPluginImage      string = "quay.io/k8scsi/hostpathplugin:v0.2.0"
-	csiExternalAttacherImage    string = "quay.io/k8scsi/csi-attacher:v0.2.0"
-	csiExternalProvisionerImage string = "quay.io/k8scsi/csi-provisioner:v0.2.1"
-	csiDriverRegistrarImage     string = "quay.io/k8scsi/driver-registrar:v0.2.0"
-)
+var csiImageVersions = map[string]string{
+	"hostpathplugin":   "v0.2.0",
+	"csi-attacher":     "v0.2.0",
+	"csi-provisioner":  "v0.2.1",
+	"driver-registrar": "v0.2.0",
+}
+var csiImageVersion string
+var csiImageRegistry string
+
+func init() {
+	flag.StringVar(&csiImageVersion, "csiImageVersion", "", "overrides the default tag used for hostpathplugin/csi-attacher/csi-provisioner/driver-registrar images")
```

Comment:
how would one specify different image versions for different images (like what's currently listed in 42-45 lines above)?